.github/workflows/test.yaml

```diff
@@ -37,6 +37,6 @@ jobs:
         run: pip install poetry
       - run: poetry config virtualenvs.create false
```

Comment:
Does this line need to remove the - as well?

Reply:
Nope! I just didn't give a particular name for that step.

Reply:
The error was that there was an orphaned `- name` because of the extra "-" in front of the following `run` section.